gateway/ln-gateway/src/lib.rs

```diff
@@ -898,6 +910,15 @@ impl Gateway {
             self.client_builder
                 .save_config(gw_client_cfg.clone(), dbtx)
                 .await?;
+            let scids: Vec<u64> = self
+                .scid_to_federation
+                .read()
+                .await
+                .keys()
+                .cloned()
+                .collect();
+
+            lnrpc.update_fedimint_scids(scids).await?;
```

Comment:
I think we should log and swallow the error here, instead of propagating it up. The reason being since if we fail to update the scids here, everything should still work, we just won't get the optimization right away. But propagating the error up will fail the registration, which I don't think we want

Reply:
updated to warn and swallow